neqo-transport/src/cc/new_reno.rs

```diff
@@ -29,16 +29,25 @@ impl Display for NewReno {
 }
 
 impl WindowAdjustment for NewReno {
-    fn on_packets_acked(&mut self, mut curr_cwnd: usize, mut acked_bytes: usize) -> (usize, usize) {
-        if acked_bytes >= curr_cwnd {
-            acked_bytes -= curr_cwnd;
-            curr_cwnd += MAX_DATAGRAM_SIZE;
-            qinfo!([self], "congestion avoidance += {}", MAX_DATAGRAM_SIZE);
-        }
-        (curr_cwnd, acked_bytes)
+    fn on_packets_acked(
+        &mut self,
+        curr_cwnd: usize,
+        _acked_bytes: usize,
+        _min_rtt: Duration,
+        _now: Instant,
+    ) -> usize {
+        curr_cwnd
```

Comment:
OK, I clearly don't understand what is going on here, because the same function for Cubic returns a value that is limited to two datagrams.